app/controllers/Admin.scala

```diff
@@ -207,5 +212,100 @@ object Admin extends Controller with PanDomainAuthActions {
     )
   }
 
+  /*
+    Section Tag association
+   */
+
+  val addSectionTagForm = Form(
+    mapping(
+      "section_id" -> longNumber,
+      "tag_id" -> nonEmptyText
+    )(assignTagToSectionFormData.apply)(assignTagToSectionFormData.unapply)
+  )
+
+  val removeSectionTagForm = Form(
+    mapping(
+      "section_id" -> longNumber,
+      "tag_id" -> nonEmptyText
+    )(unAssignTagToSectionFormData.apply)(unAssignTagToSectionFormData.unapply)
+  )
+
+  def sectionsAndTags(selectedSectionIdOption: Option[Long]) = (AuthAction andThen WhiteListAuthFilter).async {
+
+    val selectedSectionOptionFt:Future[Option[Section]] = selectedSectionIdOption match {
```

Comment:
Just a thought, but since you're explicitly declaring the type here, and there's no `selectedSection` val, you probably don't need to include the OptionFt bit in the variable name? Just a styling thought...